Manito/Manito/Screens/Main/MainViewController.swift

```diff
@@ -132,14 +132,24 @@ class MainViewController: BaseViewController {
         let alert = UIAlertController(title: "새로운 마니또 시작", message: nil, preferredStyle: UIAlertController.Style.actionSheet)
         
         let createRoom = UIAlertAction(title: "방 생성하기", style: .default, handler: nil)
-        let enterRoom = UIAlertAction(title: "방 참가하기", style: .default, handler: nil)
-        let cancel = UIAlertAction(title: "취소", style: .default, handler: nil)
+        let enterRoom = UIAlertAction(title: "방 참가하기", style: .default, handler: { _ in self.presentParticipateRoomViewController() })
+        let cancel = UIAlertAction(title: "취소", style: .cancel, handler: nil)
         
         alert.addAction(createRoom)
         alert.addAction(enterRoom)
         alert.addAction(cancel)
         present(alert, animated: true, completion: nil)
     }
+
+    func presentParticipateRoomViewController() {
+        let storyboard = UIStoryboard(name: "ParticipateRoom", bundle: nil)
+        let ParticipateRoomVC = storyboard.instantiateViewController(identifier: "ParticipateRoomViewController")
```

Comment:
p5;

```suggestion
        let ParticipateRoomVC = storyboard.instantiateViewController(identifier: ParticipateRoomViewController.class)
```
이렇게 쓸 수 있어요!

Reply:
[2244408](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/81/commits/2244408d7249c0638b248c5c62ed7819707ce5a9)
완료!